src/app/pages/storage/volumes/volume-list-controls/volumes-list-controls.component.html

```diff
@@ -1,24 +1,13 @@
 <div *ngIf="entity && conf" fxFlex class="entity-table-controls" fxLayout="row wrap" fxLayoutAlignGap="16px" fxLayoutAlign="end center">
 
   <!-- SEARCH -->
-  <!-- TODO: Replace with ix-input when its ready -->
```

Comment:
We have another place with the same text and it could be potentially affected by a similar issue. Just FYI
https://github.com/truenas/webui/blob/b0a8e1fd572bb74a22a5b1d595f0434e36790f7c/src/app/pages/common/entity/entity-table/entity-table-add-actions/entity-table-add-actions.component.html#L4

Reply:
There was a collision between two form modules. I think it should be done when fully refactoring the component.